### ISSUE TITLE
How to handle the amount of saved images?

### ISSUE THREAD

**Issue by rschmidt-webit** [Visual-Regression-Tracker/Visual-Regression-Tracker#428]
Hey Guys,

wer are using VRT with Cypress and it works well. But we run into the issue, that the storage is at its limit. 
At the moment I cant see a way to handle the saved pictures from the backend. I tried limiting the Build Numbers expecting to remove the assciated images from the deleted builds, but that did not happen.
Is there an option to limit the history on an image or some other option? Could not find anything in the docs.

Maybe some more info on the setup. We do not use branches. We only have a baseline branch and every run is tests against it and maybe updates it. Was the easiest setup and does what we need.

Thanks in Advance.

**Comment by pashidlos** [comment]
@rschmidt-webit could you provide the numbers in your project settings?
by default its 100 builds that should live 30 days 
baseline images are not erased though 
image should be erased from disc when project/build/test run is removed manually

**Comment by rschmidt-webit** [comment]
Yes we had the defaults. I changed it to 5/6 builds and 30 days. (So I could also change the branch lifetime easily and it wont delete the baseline image?)

I deleteted a lot of builds manually, but it seemed to not delete any images. I deleted than the variations, as they hat a lot of old stuff. That worked.

But in generel, if I set the max builds to 5 or six and the lifetime maybe to 3 weeks. It will delete all images form a build that gets removed trough the limit and in will delete all images after 3 weeks?
